doc/try_new_features_before_release.md

```diff
@@ -29,4 +29,25 @@ Step 1. Open a PR no matter it is merged or not. Click "Checks" tab.
 
 ![](assets/5.PNG)
 
-Step 2-4. Follow the same steps of tutorial for commit.
\ No newline at end of file
+Step 2-4. Follow the same steps of tutorial for commit.
+
+## When Azure Python SDK is not released
+
+When a feature is in private preview stage, the corresponding Azure Python SDK cannot be released. But you can still try the new feature.
+
+Click pypi and download all `.whl` files in artifact page. Install it along with the Azure Python SDK. The SDK can be retrieved from Azure SDK team. You can request a release of SDK in https://portal.azure-devex-tools.com/. Since it's in private preview, it cannot be released. You can get a private build instead.
```

Comment:
it will be great if there's some screencut.

Reply:
Sure. The artifact page is the same as previous screenshot. I can add more details.